### ISSUE TITLE
[TTS] Use turn:lanes on TTS

### ISSUE THREAD

**Issue by matheusgomesms** [organicmaps/organicmaps#5034]
Since turn lane guidance is being developed, and I know this is probably very advanced enhancement, I am opening this for future reference (and maybe implementation hehe). I could not find an exact issue on this, sorry if it's duplicated.

I would like to have lane guidance on TTS. For example (I am not English native but it should be something like this): _Stay on third lane to exit to Highway 123_ or _Stay on the two lanes to the left_.

This is very useful on highways, where you can have multiples exits or directions at the same time. On this example, one going to Santa Monica has to stay on the fifth lane only. GMaps says something like _Take the exit 53B, on direction to Interstate 10 West, and stay on the second lane to the right_.

![image](https://user-images.githubusercontent.com/86851490/234060213-4272e039-fdb9-40dd-8106-b27f38e731a2.png)

Kind of related issues: #849 #4873 #2778 #3130 (a good challenge, @zyphlar!)

**Comment by lapaz17** [comment]
@Nic787  are you workin on it?

**Comment by Nic787** [comment]
> 

No I'm not a dev, but I just love this idea.

**Comment by zyphlar** [comment]
This will definitely interact with my PR for street/highway/exit name TTS and will depend on whether the routing layer provides the information in the right places. Fortunately lane guidance is now visually present so it shouldn't be too hard when the time comes. Anyone who does work on this before me, let's coordinate or collaborate so that my TTS stuff doesn't conflict.

**Comment by RicoElectrico** [comment]
FYI my friend told me that Apple Maps does this well, if anyone needs inspiration to base upon.

**Comment by zyphlar** [comment]
@RicoElectrico if anyone uses Apple Maps and can write down the phrases it uses that could be helpful, sometimes it's hard to figure out what users are familiar with and comparing what else is out there makes it easier to decide what we should do. I have most other nav apps but not Apple.

**Comment by matheusgomesms** [comment]
> @RicoElectrico if anyone uses Apple Maps and can write down the phrases it uses that could be helpful, sometimes it's hard to figure out what users are familiar with and comparing what else is out there makes it easier to decide what we should do. I have most other nav apps but not Apple.

Do you still need these strings? I can try to write them down (both Apple and Google Maps). I agree, Apple does that well.

**Comment by euf** [comment]
@matheusgomesms  Yes please, documenting these phrases with their respective cases would be really helpful!
